ingestion-beam/src/test/java/com/mozilla/telemetry/aet/DecryptAetIdentifiersTest.java

```diff
@@ -221,4 +221,32 @@ public void testErrorOutput() throws Exception {
     pipeline.run();
   }
 
+  @Test
+  public void testTelemetryErrorOutput() throws Exception {
+    // minimal test for throughput of a single document
+    ValueProvider<String> metadataLocation = pipeline
+        .newProvider(Resources.getResource("account-ecosystem/metadata-local.json").getPath());
+    ValueProvider<Boolean> kmsEnabled = pipeline.newProvider(false);
+
+    List<String> input = ImmutableList.of("{}", "{\"payload\":{}}",
+        "{\"payload\":{\"ecosystemClientId\":\"foo\",\"ecosystemDeviceId\":\"bar\"}}");
```

Comment:
Another test case might be a non-empty payload with a disallowed field like `{"payload": {"ecosystemAnonId": "foobar"}}` which is sent to the error stream with a redacted value.

Reply:
Good call. Adding this case exposed an error in the schema for telemetry which is now fixed in this PR.